src/common/config_opts.h

```diff
@@ -913,6 +913,8 @@ SAFE_OPTION(rocksdb_db_paths, OPT_STR, "")   // path,size( path,size)*
 OPTION(rocksdb_log_to_ceph_log, OPT_BOOL, true)  // log to ceph log
 OPTION(rocksdb_cache_size, OPT_U64, 128*1024*1024)  // default rocksdb cache size
 OPTION(rocksdb_cache_shard_bits, OPT_INT, 4)  // rocksdb block cache shard bits, 4 bit -> 16 shards
+OPTION(rocksdb_compressed_cache_size, OPT_U64, 128*1024*1024)  // default rocksdb compressed block cache size
+OPTION(rocksdb_compressed_cache_shard_bits, OPT_INT, 4)  // rocksdb compressed block cache shard bits, 4 bit -> 16 shards
```

Comment:
It is similar as rocksdb_cache_shard_bits, please get more detail from https://github.com/facebook/rocksdb/wiki/Block-Cache, which made it very clear